### ISSUE TITLE
Where is the docker image on docker hub?

### ISSUE THREAD

**Issue by JonasT** [NodeBB/NodeBB#6096]
Is there a build docker image on docker hub? I found this one: https://hub.docker.com/r/nodebb/docker/ but it looks like it hasn't been updated in a long time.

Also, it looks like you're launching `node` directly as PID 1 inside the docker image. This is a problematic approach and can lead to issues with zombie processes and other breakages when any sort of subprocess is ever spawned (e.g. to run a shell command from inside node) - you should consider wrapping it as described here by adding tini right into the image: https://github.com/krallin/tini#using-tini
Basically all you need to do is:

```
ENV TINI_VERSION v0.16.1
ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
RUN chmod +x /tini
ENTRYPOINT ["/tini", "--"]

CMD [ "./nodebb", "start" ]
```
instead of the current `CMD ./nodebb start`

**Comment by julianlam** [comment]
Yes, the published image is quite old... can you give [this one a try](https://hub.docker.com/r/benlubar/nodebb/)? If it works we'll talk to @BenLubar and see if he can manage the official one for us :smile: 

https://hub.docker.com/r/benlubar/nodebb/

**Comment by BenLubar** [comment]
@julianlam do you have access to the nodebb organization on Docker Hub? When I asked 2 years ago, nobody knew who owned it.

**Comment by julianlam** [comment]
Checking...

**Comment by julianlam** [comment]
@BenLubar It looks like the nodebb org owns it, of which I am a member. At the moment trying to figure out how to add you as a collaborator.

**Comment by julianlam** [comment]
@JonasT The image on https://hub.docker.com/r/nodebb/docker/ is now updated.

**Comment by JonasT** [comment]
Thanks <3 @julianlam I tested it, it does indeed appear to be updated now
